File: elementary_computer_science/C#/random_gen.cs
using System;
namespace Creative1 {
	class RandomGen {
		static void Main(string[] args) {
			Show(Gen(200));
			Console.WriteLine("\n Fini ");
			Console.ReadLine();
		}

		static int [] Gen(int n) {
			int[] a = new int[n];
			Random r = new Random();
			for (int i = 0; i < n; ++i)
				a[i] = r.Next(n);
			return a;
		}

		static void Show(int[] s) {
			foreach (int x in s) Console.Write(x + " ");
	}
}

File: elementary_computer_science/Pascal/random_gen.pas
program randomGen;
{$B-}
uses crt;
const MN = 500;
var a: array [1..MN] of integer;
	n: integer;

procedure gen(m: integer);
var i: integer;
begin
	randomize;
	n := m;
	for i := 1 to n do a[i] := random(m);
end;

procedure display(n: integer);
var i: integer;
begin
	for i := 1 to n do write(a[i]:4);
end;

begin
	read(n);
	gen(n);
	display(n);
	writeln;
end.